core4/api/v1/request/standard/template/help.html

```diff
@@ -1,283 +1,273 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
-    <title>HELP {{ handler.__class__.__name__ }}</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <link rel="stylesheet" type="text/css" href="{{ default_static('bootstrap-material-design.custom.css') }}">
-    <link rel="stylesheet" type="text/css" href="{{ default_static('widget.css') }}">
-    <!-- <link rel="stylesheet" type="text/css" href="..\..\_static\bootstrap-material-design.custom.css"> -->
-    <!--<link rel="stylesheet" type="text/css" href="..\..\_static\widget.css">-->
-
+    <title>Download</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Not needed- we are using the mdi icons - this line imports md icons: https://vuetifyjs.com/en/customization/icons